src/services/projectVerificationFormService.ts

```diff
@@ -32,6 +32,30 @@ import { errorMessages } from '../utils/errorMessages';
 import { ProjectVerificationUpdateInput } from '../resolvers/types/ProjectVerificationUpdateInput';
 import { removeUndefinedFieldsFromObject } from '../utils/utils';
 
+const updateLastStep = async (params: {
+  projectVerificationForm: ProjectVerificationForm;
+  step: string;
+}): Promise<ProjectVerificationForm> => {
+  const { step, projectVerificationForm } = params;
+  if (
+    step === PROJECT_VERIFICATION_STEPS.PERSONAL_INFO &&
```

Comment:
Can you skip steps or frontend validates? Maybe this validation of email_confirmed, should be present in future steps or in the submit step too.

Reply:
Just in case, of some wacky url manipulation.

Reply:
Thanks for explaning me by DM :D

Reply:
Thank you for paying attention to details and security issues